src/components/ListItem/ListItemWithImage.js

```diff
@@ -506,15 +508,15 @@ const Addon = (props: AddonProps) => {
           color={colors.secondaryText}
           margin={0}
           icon="close"
-          fontSize={fontSizes.regular}
+          fontSize={fontSizes.big}
```

Comment:
Hm, there's something wrong with those icons. They get cut and neither X nor tick is shown properly